webapp/src/components/Mana/Mana.js

```diff
@@ -21,7 +21,7 @@ export default class Mana extends React.PureComponent {
   getLocalizedAmount() {
     const { amount } = this.props
 
-    if (amount) {
+    if (typeof amount === 'number') {
```

Comment:
What if we check for `amount > 0` to avoid a possible future bug?

Reply:
`>=`, 0 is a valid balance